### PR TITLE
[Network] `az network vwan create/upate`: Remove the stale argument --vnet-to-vnet-traffic

### PR DESCRIPTION
**Pull request by jsntcy**
Remove the stale argument --vnet-to-vnet-traffic for the commands `az network vwan create/upate`

The issue is https://github.com/Azure/azure-cli/issues/14772

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

File: src/virtual-wan/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+0.2.1
+++++++
+* `az network vwan create/update`: Remove the stale argument `--vnet-to-vnet-traffic`.
```
Comment (MyronFanQiu) [inline src/virtual-wan/HISTORY.rst:8]: How about adding `breaking change` here?

**Review by MyronFanQiu — APPROVED** [review]

**Comment by yonzhan** [comment]
Network

File: src/virtual-wan/HISTORY.rst (hunk 1 above)
Comment (jsntcy) [inline src/virtual-wan/HISTORY.rst:8]:
```suggestion
* [BREAKING CHANGE] `az network vwan create/update`: Remove the stale argument `--vnet-to-vnet-traffic`.
```

Comment (jsntcy) [inline src/virtual-wan/HISTORY.rst:8]:
```suggestion
* [BREAKING CHANGE] `az network vwan create/update`: Remove the stale argument `--vnet-to-vnet-traffic`.
```

**Pull request merged**